### PR TITLE
Finished second translation of spanish version

### PR DESCRIPTION
**Pull request by hamfree**
Hi, Suman!

Here you are the second translation of spanish version.

Regards!

### CODE REVIEW

File: es/jakarta.jpg
Comment (sumn2u) [inline es/jakarta.jpg:1]: Do we really need this file?

File: es/SUMMARY.md
Comment (sumn2u) [inline es/SUMMARY.md:1]: Nice work 👏

**Review by sumn2u — CHANGES_REQUESTED** [review]
Great job! I have a quick question about `jakarta.jpg`. Do we really need that file?

File: es/jakarta.jpg
Comment (hamfree) [reply es/jakarta.jpg:1]: No. 
I think that file belongs to other project and it's my fault. You can delete it!

Comment (sumn2u) [reply es/jakarta.jpg:1]: Ok. Thanks for the confirmation.

Comment (hamfree) [reply es/jakarta.jpg:1]: I've summited a new change with the file 'jakarta.jpg' deleted

File: es/SUMMARY.md
Comment (hamfree) [reply es/SUMMARY.md:1]: Thank you for your great work. That's why I chose to collaborate with your project. You explain very well for developers starting out with the JavaScript language.

**Review by sumn2u — APPROVED** [review]
LGTM 🚀

**Pull request merged**